packages/ckeditor5-image/src/imageresize.js

```diff
@@ -60,14 +60,15 @@ export default class ImageResize extends Plugin {
 /**
  * The resize options.
  *
- * Each option should have its `name`, which is a component name definition that will be
- * used in the {@link module:image/imageresize/imageresizeui~ImageResizeUI} plugin.
- * Other properties like `label` and `value` define the following:
- * a text label for the option button and the value that will be applied to the image's width.
+ * Each option should have its own `name`, which is a component name definition that will be
+ * used in the {@link module:image/imageresize/imageresizeui~ImageResizeUI} plugin and `value` which will
+ * be applied to the image's width during the {@link module:image/imageresize/imageresizecommand~ImageResizeCommand} execution.
  *
  * The value property is combined with the `resizeUnit` (`%` by default), eg: `value: '50'` and `resizeUnit: '%'` is `50%`.
  *
- * **NOTE:** If you want to set an option that will reset image to its original size, you need to pass a `null` value
+ * **NOTE:** There is additional mandatory property — `icon`, but it's necessary only for setting the standalone buttons configuration.
```

Comment:
Let's move this paragraph next to

> If you want to have separate buttons for each option \[...\]

because it's the only case when it matters. Or we may not even need this paragraph because it's already explained

> Please keep in mind that this time \*\*you should define additional \`icon\` property\*\*: